oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+=============================
+OEP-0055: Project Maintainers
+=============================
+
+.. This OEP template is based on Python's PEP standard.
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - Link to the doc in the following format::
+
+        :doc:`OEP-0055 <oep-0055-proc-Project Maintainers>`
+
+        * <XXXX is the next available OEP number>
+        * <YYYY is the abbreviated Type: proc | bp | arch>
+        * <ZZZZ is a brief (< 5 words) version of the title>
+
+   * - Title
+     - Project Maintainers
+   * - Last Modified
+     - 2022-01-28
+   * - Authors
+     - Edward Zarecor <e0d@tcril.org>
+   * - Arbiter
+     - <Arbiter's real name and email address>
+   * - Status
+     - Draft
+   * - Type
+     - Process
+   * - Created
+     - 2022-01-28
+   * - Review Period
+     - <start - target end dates for review>
+   * - Resolution
+     - <links to any discussions where the final status was decided>
+   * - References
+     - <links to any other relevant discussions or relevant related materials>
+
+Abstract
+========
+
+The Open edX platform is a service oriented platform for authoring and
+delivering online learning based on the `Django framework`_.  At the
+highest level, the platform is composed of numerous services (IDAs
+[#]_), micro-front ends (MFEs [#]_) and libraries.  The code for these
+services and libraries is managed in git and hosted on GitHub
+platform.
+
+Each component part of the Open edX platform requires maintenance to
+ensure that it meets, and continues to meet, the project definition of
+high quality.  That definition will include project standards,
+appropriate documentation, and fitness for use, among other things.
+That definition will evolve as project needs do and we learn what
+works and what doesn't.
+
+Component maintenance is work.  To ensure that that work is done, a
+person or persons must be recognized as responsible for doing it,
+and held accountable for its performance.
+
+The Open edX project, like many projects, calls this role
+"maintainer."  The notion of a project maintainer is typical of
+open-source projects and widely considered an best practice [#]_.
+
+The maintainer role is a leadership role that encompasses both
+technical and community elements.  Both dimensions of the role are
+equally important.
+
+All, non-deprecated, components of the Open edX project must have a
+named maintainer.  When no such maintainer exists, this will be
+considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in production [#]_.
```

Comment:
`in production` is vague, maybe `their performance in any community member's production environment`?

Reply:
@sarina can you resolve if my update meets muster?

Reply:
Looks good, thanks